core-library/src/main/java/org/silverpeas/core/admin/service/OrganizationController.java

```diff
@@ -368,13 +373,16 @@ static OrganizationController get() {
    * silverpeas) available in silverpeas given a userId and a componentNameRoot
    *
    * @author dlesimple
+   * @deprecated please use {@link #getAllComponentIds(String)} or
+   * {@link #getAllComponentIdsRecur(String)}
    * @param sSpaceId
    * @param sUserId
    * @param sComponentRootName
    * @param inCurrentSpace
    * @param inAllSpaces
    * @return Array of componentsIds
    */
+  @Deprecated
   String[] getAllComponentIdsRecur(String sSpaceId, String sUserId, String sComponentRootName,
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1133)